### ISSUE TITLE
plugin shows error pop-up when I open package-lock.json file

### ISSUE THREAD

**Issue by jacky96623** [aws-cloudformation/cfn-lint-visual-studio-code#306]
Steps to reproduce in VS Code:
 - create a new directory
 - run `npm init`
 - run `npm i dotenv`
 - browse the `package-lock.json` file created

Result:
![Screenshot_20230412_121916](https://user-images.githubusercontent.com/37379550/231348726-c96d7a3c-1942-49f4-a12b-10cbadcaa672.png)

Output:
```log
[Error - 12:18:07 PM] Request textDocument/documentSymbol failed.
Error: name must not be falsy
	at Function.validate (/home/jksng/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/out/vs/workbench/api/node/extensionHostProcess.js:97:25587)
	at new u (/home/jksng/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/out/vs/workbench/api/node/extensionHostProcess.js:97:25825)
	at asSymbolInformation (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:593:15)
	at convertBatch (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:193:25)
	at Object.map (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:202:17)
	at Object.asSymbolInformations (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:558:22)
	at _provideDocumentSymbols (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/documentSymbol.js:86:76)
	at async t.provideDocumentSymbols (/home/jksng/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/out/vs/workbench/api/node/extensionHostProcess.js:101:39308)
Don't believe this is a CloudFormation template. file:///home/jksng/workspace/tmp/package-lock.json. If it is please add AWSTemplateFormatVersion: '2010-09-09' (YAML) or "AWSTemplateFormatVersion": "2010-09-09" (JSON) into the root level of the document.
[Error - 12:18:07 PM] This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Don't believe this is a CloudFormation template. file:///home/jksng/workspace/tmp/package-lock.json. If it is please add AWSTemplateFormatVersion: '2010-09-09' (YAML) or "AWSTemplateFormatVersion": "2010-09-09" (JSON) into the root level of the document.".
[Error - 12:19:06 PM] Request textDocument/documentSymbol failed.
Error: name must not be falsy
	at Function.validate (/home/jksng/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/out/vs/workbench/api/node/extensionHostProcess.js:97:25587)
	at new u (/home/jksng/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/out/vs/workbench/api/node/extensionHostProcess.js:97:25825)
	at asSymbolInformation (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:593:15)
	at convertBatch (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:193:25)
	at Object.map (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:202:17)
	at Object.asSymbolInformations (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:558:22)
	at _provideDocumentSymbols (/home/jksng/.vscode-server/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/documentSymbol.js:86:76)
	at async t.provideDocumentSymbols (/home/jksng/.vscode-server/bin/b7886d7461186a5eac768481578c1d7ca80e2d21/out/vs/workbench/api/node/extensionHostProcess.js:101:39308)
Don't believe this is a CloudFormation template. file:///home/jksng/workspace/tmp/package-lock.json. If it is please add AWSTemplateFormatVersion: '2010-09-09' (YAML) or "AWSTemplateFormatVersion": "2010-09-09" (JSON) into the root level of the document.
[Error - 12:19:07 PM] This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Don't believe this is a CloudFormation template. file:///home/jksng/workspace/tmp/package-lock.json. If it is please add AWSTemplateFormatVersion: '2010-09-09' (YAML) or "AWSTemplateFormatVersion": "2010-09-09" (JSON) into the root level of the document.".
```

Expected Behaviour:
Plugin does not show pop-up if the file is found to be non-template.

Supplementary Info:
Indeed some promise rejection error is also logged when you open the package.json file, but the pop-up will not show.

**Comment by hakanson** [comment]
I am also getting this same error editing a `[redacted].json` file which is mapped to a custom JSON schema validator through another VS Code extension.   `cfn-lint --version` is `cfn-lint 0.77.0`

```
[Error - 7:37:01 AM] Request textDocument/documentSymbol failed.
Error: name must not be falsy
	at u.validate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:96:25587)
	at new u (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:96:25825)
	at asSymbolInformation (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:593:15)
	at convertBatch (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:193:25)
	at Object.map (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:202:17)
	at Object.asSymbolInformations (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:558:22)
	at _provideDocumentSymbols (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/documentSymbol.js:86:76)
	at async t.provideDocumentSymbols (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:100:39308)
[Error - 7:37:02 AM] Request textDocument/documentSymbol failed.
Error: name must not be falsy
	at u.validate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:96:25587)
	at new u (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:96:25825)
	at asSymbolInformation (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:593:15)
	at convertBatch (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:193:25)
	at Object.map (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/utils/async.js:202:17)
	at Object.asSymbolInformations (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/protocolConverter.js:558:22)
	at _provideDocumentSymbols (/Users/kevhak/.vscode/extensions/kddejong.vscode-cfn-lint-0.24.3/client/node_modules/vscode-languageclient/lib/common/documentSymbol.js:86:76)
	at async t.provideDocumentSymbols (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:100:39308)
```

**Comment by kddejong** [comment]
Should be fixed in v0.24.4
